### ISSUE TITLE
Phyton DLL Error

### ISSUE THREAD

**Issue by Imiona** [Jonius7/SteamUI-OldGlory#39]
I get a Python Error when executing the .exe File with the newest Version (5.8.1)
![error](https://user-images.githubusercontent.com/11210717/160649115-5c956176-17fe-4317-b1f9-fd4082495b64.png)

I downloaded the previous Version (5.7.4.2) and it starts up fine.

So something has changed.

**Comment by EternityEngine** [comment]
I get the same error, except it's preceded by a different one:
![image](https://user-images.githubusercontent.com/20052973/161461092-ebfbafff-7587-4b67-97e1-1f2992a4fc43.png)

I had this first error with another program a while back and it's because Windows 7 doesn't have the DLL it's complaining about (it's a system file in later versions of Windows). I tried the fix I found (installing [this](https://github.com/nalexandru/api-ms-win-core-path-HACK), had to be in System32 instead of the program folder this time), and the program then opened without error.

So basically Old Glory is by default no longer compatible with Windows 7.

**Comment by Jonius7** [comment]
@EternityEngine At the bottom of Troubleshooting:
https://github.com/Jonius7/SteamUI-OldGlory#troubleshooting

**Comment by EternityEngine** [comment]
@Jonius7 That is a different DLL file, and I have both the x86 and x64 versions of that redist installed (though it is labelled 2015-2022 instead). My versions were slightly out of date, so I tried reinstalling the redist and removing the DLL from System32, and I got the same error again.

**Comment by Jonius7** [comment]
I think the issue is I am using Python 3.9 now (which does not support Windows 7)
I will switch back to 3.8 at some point in the future

**Comment by Jonius7** [comment]
Latest version 5.8.3 is built using Python 3.8 now

**Comment by Imiona** [comment]
Works now, thx Jonius
